test/SparqlAntlrParserTest.cpp

```diff
@@ -1799,7 +1799,7 @@ TEST(SparqlParser, FunctionCall) {
   expectFunctionCall(absl::StrCat(xsd, "double>(?x)"),
                      matchUnary(&makeConvertToDoubleExpression));
   expectFunctionCall(absl::StrCat(xsd, "decimal>(?x)"),
-                     matchUnary(&makeConvertToDoubleExpression));
+                     matchUnary(&makeConvertToDecimalExpression));
```

Comment:
Obviously there also need to be tests inside the `SparqlExpressionTest.cpp` file to test for the semantics, and here you should also test the `convertToStrExpression`.